src/media/js/routes.js

```diff
@@ -60,6 +60,8 @@ define('routes',
         'recommended': '/api/v2/apps/recommend/?cache=1',
         'record_free': '/api/v2/installs/record/',
         'record_paid': '/api/v2/receipts/install/',
+        'region': '/api/v2/services/region/{0}/?cache=86400&vary=0',
```

Comment:
Sadly can't use the CDN or be offline cached yet, because of https://bugzilla.mozilla.org/show_bug.cgi?id=1162061